### PR TITLE
[feat-5431]: Improve accessibility IncidentDetail

### PR DESCRIPTION
**Pull request by OscarBakker**
Ticket: [SIG-5431](https://gemeente-amsterdam.atlassian.net/browse/SIG-5431)

## Signalen

Before opening a pull request, please ensure:

- Make sure your PR title follows naming conventions: [feat-1234]: name feature
- Double-check your branch is based on `main` and targets `main`
- Pull request has tests (we are going for 100% coverage!)
- Code is well-commented, linted and follows project conventions
- Committed source code is headed by the correct SPDX license expression

Be kind to code reviewers, please try to keep pull requests as small and focused as possible :)

### CODE REVIEW

File: src/signals/incident-management/containers/IncidentDetail/components/Detail/components/Contact/Contact.tsx [hunk 1]
```diff
@@ -161,7 +163,7 @@ export const Contact = ({ incident, showPhone }: Props) => {
               variant="application"
             />
           )}
-          {showPhone ? (
+          {showPhone && incident.reporter.phone ? (
```
Comment (vdegraaf) [inline src/signals/incident-management/containers/IncidentDetail/components/Detail/components/Contact/Contact.tsx:166]: How is this improve accessibility?

File: src/signals/incident-management/containers/IncidentDetail/components/Detail/components/Contact/Edit.tsx [hunk 2]
```diff
@@ -72,6 +73,7 @@ const Edit = ({ onClose, incident, submit }: Props) => {
         <div>
           <StyledInput
             {...register('email')}
+            id={'email'}
```
Comment (vdegraaf) [inline src/signals/incident-management/containers/IncidentDetail/components/Detail/components/Contact/Edit.tsx:76]: For my knowledge: why should an input have an id?

File: src/signals/incident-management/containers/IncidentDetail/components/Detail/components/Contact/Contact.tsx (hunk 1 above)
Comment (OscarBakker) [reply src/signals/incident-management/containers/IncidentDetail/components/Detail/components/Contact/Contact.tsx:166]: cause you can't have an empty link on a page

File: src/signals/incident-management/containers/IncidentDetail/components/Detail/components/Contact/Edit.tsx (hunk 2 above)
Comment (OscarBakker) [reply src/signals/incident-management/containers/IncidentDetail/components/Detail/components/Contact/Edit.tsx:76]: a label in associated with an input by either nesting the input, or <label for='naam'>name</label> <input id='naam' asdfasfa />

**Review by SireeKoolenWijkstra — APPROVED** [review]
looks good to me, behalve dan dat de links rechtsboven niet accessable zijn (werd getriggerd door de melding dat lege links niet accessable zijn): 
image

**Comment by OscarBakker** [comment]
> looks good to me, behalve dan dat de links rechtsboven niet accessable zijn (werd getriggerd door de melding dat lege links niet accessable zijn):
> image

some of the asc components have some a11Y issues

**Pull request merged**